cmd/build.go

```diff
@@ -44,28 +46,27 @@ var buildCmd = &cobra.Command{
 		}
 
 		syscalls := make([]string, 0)
+		var syscallList = make(map[string]int)
 		for _, fileObj := range files {
-			//fmt.Println("[" + fileObj.Name() + "]")
-
-			file, err := os.Open(inputDirectory + "/" + fileObj.Name())
+			file, err := os.Open(filepath.Join(inputDirectory, fileObj.Name()))
 			if err != nil {
-				return fmt.Errorf("error opening file %s: %w", file.Name(), err)
+				return fmt.Errorf("error opening file: %w", err)
```

Comment:
I would have kept the filename in the error,  but why not

Reply:
you're right, I forgot to have the `fileObj.Name()` information